### DISCUSSION TITLE
[Common] Possible to adjust default screen position when using full screen borderless?

### DISCUSSION THREAD

**Discussion by pvpxan** [julianxhokaxhiu/FFNx#521] [Q&A]
Using FFNx to play Final Fantasy 7 through 7th heaven. I have multiple monitors and play with a gaming laptop hooked to an external monitor and docking station. When the game launches, it is slightly out of alignment with my screen and the task bar is visible. I have to move it to fit in place. I think it works fine when using my laptop with no external screens, but was hoping for a way to tweak the position. I am using DX 12 for the graphics API.

**Comment by julianxhokaxhiu** [comment]
Taskbar visibility is something you can configure at the OS level, there's no way to go on top unless you put it to hidden.

Regarding the starting position I remember it is calculated based on the center of the screen so could be it has an offset due to the taskbar. Feel free to change your own OS configs and try again.

**Reply by pvpxan** [reply]
Well, I do not believe it is an OS issue as I do not have this issue with other applications that run full screen windowed. Let me walk you through what is happening.
1. Launch FF7 via 7th Heaven.
2. FF7 launches and is not centered on the screen. Task bar is visible.
3. Manually move the FF7 window to be aligned with the screen.
4. When alignment is achieved, the taskbar AUTOMATICALLY becomes hidden.

Something is clearly happening here.
